tests/serializers/tests.py

```diff
@@ -353,6 +353,34 @@ def test_serialize_proxy_model(self):
         self.assertEqual(base_data, proxy_proxy_data.replace('proxy', ''))
 
 
+class SerializerAPITests(TestCase):
+
+    def test_stream_class(self):
+        class File(object):
+            def __init__(self):
+                self.lines = []
+
+            def write(self, line):
+                self.lines.append(line)
+
+            def getvalue(self):
+                return ''.join(self.lines)
+
+        class Serializer(serializers.json.Serializer):
+            stream_class = File
+
+        score = Score(score=3.4)
+        score.save()
```

Comment:
You should be able to use `SimpleTestCase` (which prevents any queries) by setting an `id` manually on your score instance, avoiding the `save()` call and passing a singleton list containing `score` to `serializer.serialize()`:

```python
data = serializer.serialize([Score(id=1, score=3.4)])
```

With this approach you should be able to hardcode the `"pk": 1` below as well.